diff --git a/Formulars/TaxPro/British Columbia/~$iOS app (excel formula) - BC.xlsx b/Formulars/TaxPro/British Columbia/~$iOS app (excel formula) - BC.xlsx
deleted file mode 100644
index 11d0ea6..0000000
Binary files a/Formulars/TaxPro/British Columbia/~$iOS app (excel formula) - BC.xlsx and /dev/null differ
diff --git a/Formulars/TaxPro/New Brunswick/~$iOS app (excel formula) - NB.xlsx b/Formulars/TaxPro/New Brunswick/~$iOS app (excel formula) - NB.xlsx
deleted file mode 100644
index 11d0ea6..0000000
Binary files a/Formulars/TaxPro/New Brunswick/~$iOS app (excel formula) - NB.xlsx and /dev/null differ
diff --git a/TaxCalculator.xcworkspace/xcuserdata/mengyiluo.xcuserdatad/UserInterfaceState.xcuserstate b/TaxCalculator.xcworkspace/xcuserdata/mengyiluo.xcuserdatad/UserInterfaceState.xcuserstate
index 5010e80..3804235 100644
Binary files a/TaxCalculator.xcworkspace/xcuserdata/mengyiluo.xcuserdatad/UserInterfaceState.xcuserstate and b/TaxCalculator.xcworkspace/xcuserdata/mengyiluo.xcuserdatad/UserInterfaceState.xcuserstate differ
diff --git a/TaxCalculator/NewBrunswickTax.swift b/TaxCalculator/NewBrunswickTax.swift
index c69a863..93f70e0 100644
--- a/TaxCalculator/NewBrunswickTax.swift
+++ b/TaxCalculator/NewBrunswickTax.swift
@@ -304,7 +304,7 @@ class NewBrunswickTax : ProvincialTax {
             }
             
         }
-        print("Deduction_2012 \(Deduction_2012) Deduction_2011\(Deduction_2011), ProportionOfNetForeignBu\(ProportionOfNetForeignBusinessIncome) FederalForeignTaxCredit \(FederalForeignTaxCredit) ProvincialForeignTaxCredit \(ProvincialForeignTaxCredit)")
+       // print("Deduction_2012 \(Deduction_2012) Deduction_2011\(Deduction_2011), ProportionOfNetForeignBu\(ProportionOfNetForeignBusinessIncome) FederalForeignTaxCredit \(FederalForeignTaxCredit) ProvincialForeignTaxCredit \(ProvincialForeignTaxCredit)")
         
         return (Deduction_2012, Deduction_2011, ProportionOfNetForeignBusinessIncome, FederalForeignTaxCredit, ProvincialForeignTaxCredit)
     
